src/pages/search/registration_search/filters/AdvancedSearchRow.tsx

```diff
@@ -58,17 +59,22 @@ export const AdvancedSearchRow = ({ removeFilter, baseFieldName }: AdvancedSearc
         {({ field }: FieldProps<string>) => (
           <TextField
             {...field}
-            fullWidth
-            sx={{ maxWidth: '30rem' }}
-            variant="outlined"
-            inputProps={{ sx: { py: '0.75rem' } }}
-            data-testid={dataTestId.startPage.advancedSearch.advancedValueField}
+            variant="standard"
+            size="small"
             label={t('search.search_term_label')}
+            data-testid={dataTestId.startPage.advancedSearch.advancedValueField}
+            fullWidth
           />
         )}
       </Field>
-      <Button onClick={removeFilter} size="small" data-testid={dataTestId.startPage.advancedSearch.removeFilterButton}>
-        {t('search.remove_filter')}
+      <Button
+        sx={{ minHeight: '36px', minWidth: '36px' }}
+        size="small"
+        variant="text"
```

Comment:
Dette ser ut som en `IconButton`?
```suggestion
      <IconButton
        size="small"
        color="primary"
```